# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hugin/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- hugin-0.5/src/include/PT/PanoramaMemento.h_orig	2006-03-04 17:49:13.582899696 +0100
+++ hugin-0.5/src/include/PT/PanoramaMemento.h	2006-03-04 17:50:22.928357584 +0100
@@ -154,7 +154,7 @@
     double getHFOV() const;
 
     /** set HFOV in degrees */
-    void Lens::setHFOV(double d);
+    void setHFOV(double d);
 
     /** get focal length of lens, it is calculated from the HFOV */
     double getFocalLength() const;
@@ -389,13 +389,13 @@
     void printScriptLine(std::ostream & o) const;
 
     /// return string name of output file format
-    static const std::string & PanoramaOptions::getFormatName(FileFormat f);
+    static const std::string & getFormatName(FileFormat f);
 
     /** returns the FileFormat corrosponding to name.
      *
      *  if name is not recognized, FileFormat::TIFF is returned
      */
-    static FileFormat PanoramaOptions::getFormatFromName(const std::string & name);
+    static FileFormat getFormatFromName(const std::string & name);
 
     /** calculate height of the output panorama
      *
--- hugin-0.5/src/include/PT/Panorama.h_orig	2006-03-04 17:50:52.478865224 +0100
+++ hugin-0.5/src/include/PT/Panorama.h	2006-03-04 17:52:29.402130648 +0100
@@ -542,7 +542,7 @@
      *
      *  only copies inherited variables
      */
-    void Panorama::updateLensToImages(unsigned int lensNr);
+    void updateLensToImages(unsigned int lensNr);
 
     /** copy lens variables to image variables.
      *  update all images that use lensNr
